pkg/tfbridge/provider.go

```diff
@@ -780,18 +780,17 @@ func (p *Provider) Check(ctx context.Context, req *pulumirpc.CheckRequest) (*pul
 	rescfg := MakeTerraformConfigFromInputs(ctx, p.tf, inputs)
 	warns, errs := p.tf.ValidateResource(ctx, tfname, rescfg)
 	for _, warn := range warns {
-		if err = p.host.Log(ctx, diag.Warning, urn, fmt.Sprintf("%v verification warning: %v", urn, warn)); err != nil {
-			return nil, err
-		}
+		GetLogger(ctx).Warn(fmt.Sprintf("%v verification warning: %v", urn, warn))
```

Comment:
Is this line intentionally changing?

Reply:
Yeah, host is nil in tests which causes this to panic. `GetLogger` seems to handle that but it's not obvious it logs to the same place. I'll revert to manually handling this for now, we can ship this later if this is appropriate.
cc @iwahbe